src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -52,11 +54,14 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
         'ctrlFForSearch',
     ];
 
+    static PRESENCE_SETTINGS = [
```

Comment:
```suggestion
    public static PRESENCE_SETTINGS = [
```

Reply:
please fix the others while we're here

Reply:
It feels like these should be private which is weird... So I think perhaps simple `const`s should work fine?

Reply:
as members of the class I don't think they can be `const`, but `private` would be better than what they currently are :)

Reply:
I meant moving them out of the class

Reply:
ah, let's not do that in this PR so the variables are consistent. Moving them all out of all the related classes would be a whole other PR.